### PR TITLE
Install standalone dipviewer, and fix install paths to match

### PR DESCRIPTION
**Pull request by wcaarls**
I tested this under Linux and Windows, but please check whether this still works under MacOS and the Windows installer.

### CODE REVIEW

File: javaio/CMakeLists.txt [hunk 1]
```diff
@@ -28,7 +28,7 @@ else()
    target_compile_definitions(DIPjavaio PUBLIC DIP__JAVAIO_IS_STATIC)
 endif()
 
-install(TARGETS DIPjavaio DESTINATION lib EXPORT DIPlibTargets)
+install(TARGETS DIPjavaio EXPORT DIPlibTargets DESTINATION ${DIP_DESTINATIONS})
```
Comment (crisluengo) [inline javaio/CMakeLists.txt]: Shouldn't this be without `DESTINATION`? `${DIP_DESTINATIONS}` already includes the keyword.

Comment (wcaarls) [reply javaio/CMakeLists.txt]: True! Fixed it.

**Comment by crisluengo** [comment]
I've set the `rpath` for the executable, the issue showed up on MacOS with my non-default install path, but I presume Linux would have the same problem if not installed to `/usr/local`.

I will poke @ronligt in case he has time to test if this affects his packaging in any way.

**Comment by wcaarls** [comment]
Forgot about that. Thanks!

**Pull request merged**